.rubocop.yml

```diff
@@ -1,3 +1,8 @@
 inherit_gem:
   rubocop-govuk:
     - config/default.yml
+
+# Renaming files from "gds-sso" to "gds_sso" would be a breaking
+# change.
+Naming/FileName:
```

Comment:
We should list the file names explicitly if we can, so this Cop can still apply to future ones.